### DISCUSSION TITLE
Full text search

### DISCUSSION THREAD

**Discussion by Aceriz** [nextcloud/all-in-one#1133] [Questions] [answered]
So I just finished doing the update for the NextcloudAIO master container and noticed that there was now the option of adding the full text search.  I went a head and added it. And spun up the containers

In the next cloud settings the apps for the full text search were fully added automatically.
When I go to the little microscope icon on the nextcloud home page for the full text search 
Nothing shows up in search results

I am wondering do I have to run an index command for my existing files ? If so how would I do that

**Comment by szaimen** [comment, marked as answer]
It should do the indexing automatically upon the first startup. Results will be displayed in the unified search.
